### ISSUE TITLE
Nothing happens when I run

### ISSUE THREAD

**Issue by Occupy4Elephants** [chaica/feed2tweet#51]
I have it all configured properly.  I've checked and reached and checked again but nothing is happening when I run any of the flags with feed2tweet, including --debug.  I'm using Kali linux as my main OS.  I am including the path to my ini...
feed2tweet --debug /home/emily/feed2tweet/feed2tweet.ini
I don't get any error messages when I hit enter.  Thanks.

**Comment by chaica** [comment]
Hi,
Could you try using the *-c* option to locate the config file ? in your case 
```
$ feed2tweet --debug -c /home/emily/feed2tweet/feed2tweet.ini
```

**Comment by Occupy4Elephants** [comment]
oh my bad....I left that out in the example I posted.  I've been using the -c option.  Sorry about that.

Ex
feed2tweet --debug -c /home/emily/feed2tweet/feed2tweet.ini

Thanks

**Comment by Occupy4Elephants** [comment]
I don't know...maybe it has something to do with my RSS feed.  I created an alert on google.  When I created the alert and previewed what it looks like, it lists all the news stories that should be included in that feed.  However after it's created, when I click the RSS feed icon on the google alerts page, the results are completely empty.  No stories like what was shown in the preview.  So I tried finding a different news to RSS feed script or app but neither of those worked either.  Could never get them verified through a 3rd party validation service that checks whether the RSS link is valid.  So I guess this is beyond the scope of your script unless you have any insight into what might be going on with my google alerts.  Thx
